typescript/examples/vercel-ai/polymarket/package.json

```diff
@@ -12,12 +12,12 @@
     "author": "",
     "license": "MIT",
     "dependencies": {
-        "@ai-sdk/openai": "^1.0.4",
+        "@ai-sdk/openai": "^1.0.11",
         "@goat-sdk/adapter-vercel-ai": "workspace:*",
         "@goat-sdk/core": "workspace:*",
         "@goat-sdk/plugin-polymarket": "workspace:*",
         "@goat-sdk/wallet-viem": "workspace:*",
-        "ai": "catalog:",
+        "ai": "4.0.22",
```

Comment:
not sure what catalog does (not even claude knows) but it was failing with O1

Reply:
Catalog is a pnpm thing, it ensures we use the same package version on all packages from the monorepo.

https://pnpm.io/catalogs